### PR TITLE
feat: added the card component

### PR DESCRIPTION
**Pull request by pradipgharge**

### CODE REVIEW

File: css/utilities.css [hunk 1]
```diff
@@ -21,4 +22,13 @@ ol {
 .container {
     width: 80%;
     margin: auto;
+}
+
+.font-light{
+    color:rgb(107, 105, 105);
```
Comment (sri-rishi) [inline css/utilities.css]: use color variable naming

File: pages/components/Cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    display: flex;
+    border-radius: inherit;
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    justify-content: center;
+
+}
+.card-hz-body-container{
+    display: flex;
+    padding: 0.8rem 0.6rem;
```
Comment (sri-rishi) [inline pages/components/Cards/cards.css]: use px for units less than 1rem

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by manish-GH — APPROVED** [review]
Add a cursor: pointer on hover for dismiss button and wishlist button. You could consider changing the on hover color for the blue button.

Everything else looks good, nice work

**Review by sri-rishi — APPROVED** [review]

**Review by manish-GH — APPROVED** [review]

**Review by Anurag021 — APPROVED** [review]
Code looks good to me.

File: pages/components/Cards/cards.css (hunk 2 above)
Comment (pradipgharge) [reply pages/components/Cards/cards.css]: Made the suggested changes

**Pull request merged**